staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -109,13 +114,19 @@ func main() {
 			}
 
 			var result Example
-			err = tprclient.Post().
+			req := tprclient.Post().
 				Resource("examples").
 				Namespace(v1.NamespaceDefault).
-				Body(example).
+				Body(example)
+			err = req.
 				Do().Into(&result)
 
 			if err != nil {
+				fmt.Println()
+				if kind_created_here {
+					glog.Infoln("Probably because of delay issue noted in https://github.com/kubernetes/features/issues/95 ...")
+				}
+				glog.Fatalf("Unable to create example1 --- request=%#v, result=%#v, err=%#v", req, result, err)
```

Comment:
I don't like that you are mixing fmt.Print() with **glog**.
The original example was only using fmt, so it would be better to stick with it, or replacing fmt with glog everywhere.